templates/plugin/rrdcached.conf.erb

```diff
@@ -22,4 +22,7 @@
 <% if @xff -%>
   XFF "<%= @xff %>"
 <% end -%>
+<% if @collectstatistics -%>
```

Comment:
please use unless like it's done here https://github.com/michakrause/puppet-module-collectd/blob/rrdcached-statistics/templates/plugin/rrdcached.conf.erb#L4-L6